src/bodypart.cpp

```diff
@@ -196,6 +196,7 @@ void body_part_type::load( const JsonObject &jo, const std::string & )
     mandatory( jo, was_loaded, "hit_size", hit_size );
     mandatory( jo, was_loaded, "hit_difficulty", hit_difficulty );
     mandatory( jo, was_loaded, "hit_size_relative", hit_size_relative );
+    mandatory( jo, was_loaded, "encumbrance_effects", encumbrance_effects );
```

Comment:
I don't think it should be mandatory since for exemple `head` deosn't have one.

Reply:
Personally, I think head should have encumbrance effects.